docs/core/preview3/tools/csproj.md

```diff
@@ -43,16 +41,16 @@ consumed.
 
 The element can contain one or more of the following values:
 
-* Compile – are the contents of the lib folder available to compile against
-* Runtime – are the contents of the runtime folder distributed
-* ContentFiles – are the contents of the contentfiles folder used
-* Build – do the props/targets in the build folder get used
-* Native – are the contents from native assets copied to the output folder for runtime
-* Analyzers – do the analyzers get used
+* Compile – are the contents of the lib folder available to compile against.
+* Runtime – are the contents of the runtime folder distributed.
+* ContentFiles – are the contents of the contentfiles folder used.
+* Build – do the props/targets in the build folder get used.
+* Native – are the contents from native assets copied to the output folder for runtime.
+* Analyzers – do the analyzers get used.
```

Comment:
"get used" is an awkward expression -- "are the props/targets in the build folder used" and "are the analyzers used"